sycl/test-e2e/Basic/submit_time.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+
+int main(void) {
+  sycl::queue queue(
+      sycl::property_list{sycl::property::queue::enable_profiling()});
+  sycl::event event = queue.submit([&](sycl::handler &cgh) {
+    cgh.parallel_for<class set_value>(sycl::range<1>{1024},
+                                      [=](sycl::id<1> idx) {});
+  });
+  auto submit =
+      event.get_profiling_info<sycl::info::event_profiling::command_submit>();
+  auto start =
```

Comment:
Is there public API that corresponds to `isGetDeviceAndHostTimerSupported`? I'd like to have an explicit call to it rather than implicit one that might or might not happen.

Reply:
There is no public API that corresponds to it. Currently isGetDeviceAndHostTimerSupported is used internally in runtime to decide how we are going to calculate "submit" timestamp - either using backend API call or using fallback implementation.

Reply:
Put it into the comment here and I'll LGTM.

Reply:
Added comments.